motr/st/utils/motr_utils_st.sh

```diff
@@ -112,193 +112,193 @@ test_with_N_K()
 	motr_service_start $N $K $S $P $stride
 	dix_init
 
-	# Test m0client utility
-	/usr/bin/expect <<EOF
-	set timeout 20
-	spawn $motr_st_util_dir/m0client $MOTR_PARAMS_V > $SANDBOX_DIR/m0client.log
-	expect "m0client >>"
-	send -- "touch $object_id3\r"
-	expect "m0client >>"
-	send -- "write $object_id2 $src_file $block_size $block_count $blks_per_io\r"
-	expect "m0client >>"
-	send -- "read $object_id2 $dest_file $block_size $block_count $blks_per_io\r"
-	expect "m0client >>"
-	send -- "delete $object_id3\r"
-	expect "m0client >>"
-	send -- "delete $object_id2\r"
-	expect "m0client >>"
-	send -- "quit\r"
-EOF
-	echo "m0client test is Successful"
+#	# Test m0client utility
```

Comment:
Why this code is commented ?

Reply:
I have used this ST to test out the m0_conf_pver_status() API, I just wanted to run m0cp once to see the output, so commented rest of the ST. This change is not intended to be merged.